charts/signoz/templates/_helpers.tpl

```diff
@@ -354,6 +354,29 @@ Return the proper otelCollector image name
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create the name of the clusterRole to use
```

Comment:
@prashant-shahi why is clusterrole needed for otel-collector pods? Is it due to logs? Otel-collector-metrics would need cluster role to scrape metrics about pods and other resources, right?

Reply:
@ankitnayan yes, it was for logs collection.
Since now it's reverted. We have removed rules and also disabled cluster role by default.
However, we are giving users option to enable it in case they need it.